sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/EventHubReactorSession.java

```diff
@@ -68,6 +79,32 @@
             messageSerializer, openTimeout, retryPolicy);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public Mono<AmqpSendLink> createProducer(
+        String linkName, String entityPath, Duration timeout, AmqpRetryPolicy retry,
```

Comment:
I think it's weird we have a new line before arguments. We should be consistent here.

Reply:
Updated.